src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -92,6 +92,32 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 	func showTanScreen() {
 		let tanInputViewModel = TanInputViewModel(
 			coronaTestService: model.coronaTestService,
+			presentOverrideTestNotice: { [weak self] testType, feedbackHandler in
+				
+				let footerViewModel = FooterViewModel(
+					primaryButtonName: AppStrings.ExposureSubmission.OverwriteNotice.primaryButton,
+					isSecondaryButtonHidden: true
+				)
+
+				let overwriteNoticeViewController = TestOverwriteNoticeViewController(
+					testType: testType,
+					didTapPrimaryButton: {
+						self?.navigationController?.dismiss(animated: true) {
+							feedbackHandler(true)
+						}
+					},
+					didTapCloseButton: { [weak self] in
+						self?.navigationController?.dismiss(animated: true) {
+							feedbackHandler(false)
+						}
+					}
+				)
+
+				let footerViewController = FooterViewController(footerViewModel)
+				let topBottomViewController = TopBottomContainerViewController(topController: overwriteNoticeViewController, bottomController: footerViewController)
+				let nc = UINavigationController(rootViewController: topBottomViewController)
```

Comment:
`TestOverwriteNoticeViewController` uses the protocol `dismissHandling` to handle that you should use `DismissHandlingNavigationController` instead.

Reply:
@KaiTeuber I need the `UINavigationController` to show a navigation bar. `TopBottomContainerViewController` is presented and not pushed.

Reply:
@freshking Just replace `UINavigationController` with `DismissHandlingNavigationController` that's all needed to respect the DismissHandling